index.html

```diff
@@ -790,6 +790,17 @@ <h2>
 Note: This parameter may not be supported by all <a>DID methods</a>.
             </td>
           </tr>
+          <tr>
+            <td>
+<code>initial-values</code>
+            </td>
+            <td>
+Some <a>DID methods</a> may require an interim period of time (e.g. block intervals) before a new identifier and its initial <a>DID document</a> 
+state is propagated within the underlying <a>DID registry</a>. To make resolution of a DID possible during this period, <a>DID methods</a> may require passage of additional values 
+to their resolver logic to ensure correct processing. <a>DID methods</a> SHOULD use the `initial-values` parameter if passage of any such values is necessary. 
+Note: This parameter may not be required by all <a>DID methods</a>.
```

Comment:
```suggestion
Note: This parameter may not be supported by all <a>DID methods</a>.
```
Or did you actually mean "required"?